bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/ExitInfoCallback.kt

```diff
@@ -35,6 +35,18 @@ internal class ExitInfoCallback(
         return true
     }
 
+    @RequiresApi(Build.VERSION_CODES.R)
```

Comment:
I think that these annotations should be lifted to the class level instead of each of the methods